23_Arithmetic2UnitTest/app_resources/testWorkgroup.comp.hlsl

```diff
@@ -14,7 +14,8 @@ typedef vector<uint32_t, config_t::ItemsPerInvocation_0> type_t;
 // final (level 1/2) scan needs to fit in one subgroup exactly
 groupshared uint32_t scratch[mpl::max_v<int16_t,config_t::SharedScratchElementCount,1>];
 
-#include "../../common/include/WorkgroupDataAccessors.hlsl"
+#include "nbl/examples/workgroup/DataAccessors.hlsl"
```

Comment:
@AnastaZIuk I need help here, the NblExamples project also needs to have its own builtins thing for HLSL headers